source/js/main.js

```diff
@@ -57,8 +57,8 @@ export default function() {
       $('li:first', $(this)).addClass('selected');
     });
 
-    $('[data-toggle="$: \'jquery\',\n' +
-        '  jQuery: \'jquery\'"]').tooltip();
+    // $('[data-toggle="$: \'jquery\',\n' +
+    //     '  jQuery: \'jquery\'"]').tooltip();
```

Comment:
Why is this commented out?

Reply:
The tooltip Plugin wasn't available by default anymore and I honestly don't know where it's used.

Reply:
Let's remove that piece of code and use your comment as a commit message then?

Reply:
Sounds link you didn't find anything on staging too. Okay, I'll create a separate commit for the removal.

Reply:
Yup. Still seems to work great :)